tests/src/test/java/com/cloudera/director/google/GoogleLauncherTest.java

```diff
@@ -59,6 +61,8 @@
 
   @BeforeClass
   public static void beforeClass() throws IOException {
+    Assume.assumeFalse(System.getProperty("GCP_PROJECT_ID", "").isEmpty());
```

Comment:
`testFixture` is only used in `testLauncher()`. Could just change it from a field to a method variable.